apps/ledger-live-mobile/src/families/osmosis/ClaimRewardsFlow/01-SelectValidator.js

```diff
@@ -0,0 +1,126 @@
+// @flow
+import invariant from "invariant";
+import React, { useCallback } from "react";
+import { View, StyleSheet, FlatList } from "react-native";
+import { useSelector } from "react-redux";
+
+import type { Transaction } from "@ledgerhq/live-common/families/cosmos/types";
```

Comment:
this should be
`import type { Transaction } from "@ledgerhq/live-common/families/osmosis/types";`

Reply:
Got it. Should this also be updated in the Redelegation flow? @JesseKuntz

Reply:
done.